<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>720</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>Generate!</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="visualLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>OutputImg</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="segmentSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #c5c8e6;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #525dbf; 
	border: 6px solid #525dbf;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>segment (supports pow(2, k) + 1) : </string>
    </property>
   </widget>
   <widget class="QLabel" name="segmentLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="roughnessSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #c5c8e6;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #525dbf; 
	border: 6px solid #525dbf;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>roughness</string>
    </property>
   </widget>
   <widget class="QLabel" name="roughnessLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="colorSchemeBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>31</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI Semilight</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>grayscale</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>normal</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="denoiseSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>280</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #c5c8e6;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #525dbf; 
	border: 6px solid #525dbf;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="denoiseLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>250</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>denoise</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>color scheme :</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImgButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>720</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>Save Image...</string>
    </property>
   </widget>
   <widget class="QSlider" name="repeatPeriodSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>580</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #c5c8e6;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #525dbf; 
	border: 6px solid #525dbf;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="segmentLabel_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>period : 1s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="repeatCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>580</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>repeat</string>
    </property>
   </widget>
   <widget class="QSlider" name="normalizeFunctionSlider">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>500</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #c5c8e6;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #525dbf; 
	border: 6px solid #525dbf;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>linear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>510</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>sig</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoSaveCheckBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>680</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>Automatically Save Image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="gaussianNoiseCheckBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>Add Gausian noise on large regions</string>
    </property>
   </widget>
   <widget class="QSlider" name="minHeightSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #b6cfd9;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #3ab7d6; 
	border: 6px solid #3ab7d6;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>min height</string>
    </property>
   </widget>
   <widget class="QLabel" name="minHeightLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxHeightLabel">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>255</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
     </font>
    </property>
    <property name="text">
     <string>max height</string>
    </property>
   </widget>
   <widget class="QSlider" name="maxHeightSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { 
	background-color: #b6cfd9;
	border: 0px solid #424242; 
	height: 16px; 
	border-radius: 4px;
}

QSlider::handle:horizontal { 
	background-color: #3ab7d6; 
	border: 6px solid #3ab7d6;
	width: 6px; 
	margin-top: -6px; 
	margin-bottom: -6px; 
	border-radius: 9px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 10px;
}</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
